package.json

```diff
@@ -54,6 +54,7 @@
     "@types/jest": "^26.0.20",
     "babel-jest": "^26.6.3",
     "codecov": "^3.6.5",
+    "detox": "^18.18.1",
```

Comment:
why move this to the root package?

Reply:
This was needed to get detox working correctly with the change of removing the flake tracker. I had tried keeping it in the mobile package.json and was having issues. Moving it to the root of the repo resolved these issues.